docs/lookout-sdk.md

```diff
@@ -40,6 +40,8 @@ To perform a `NotifyPushEvent` call to an analyzer and serve the **source{d} Loo
 $ lookout-sdk push
 ```
 
+`lookout-sdk` will exit with a non `0` exit code if the called analyzer did not responded or if it returned an error.
```

Comment:
I think in case the analyzer does not respond, the execution will not end, because we don't have any timeouts.
What about:
```suggestion
`lookout-sdk` will exit with a non `0` exit code if the gRPC call to the analyzer ends with an error.
```

Reply:
I meant: _if the analyzer could not be called_;
what about this other way?
```suggestion
`lookout-sdk` will exit with a non `0` exit code if the gRPC call to the analyzer
ends with an error or it could not be performed.
```

Reply:
To me they are the same: ends with an unreachable error code. But ok if you think it's more clear.